packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -16,6 +16,29 @@ String shortHash(Object object) {
 String describeIdentity(Object object) =>
     '${object.runtimeType}#${shortHash(object)}';
 
+/// Returns a short description of an enum value.
+///
+/// Strips off the enum class name from the `enumEntry.toString()`.
+///
+/// For example:
```

Comment:
this should use the `## Sample code` pattern so that it gets analyzed (look around some of the other files)